components/treeview/stories/treeview.stories.js

```diff
@@ -260,13 +428,17 @@ WithSections.args = {
 		},
 	],
 };
-WithSections.tags = ["autodocs", "!dev"];
-WithSections.parameters = {
+Sections.tags = ["autodocs", "!dev"];
```

Comment:
Changing this one and `WithDropTarget` since Storybook alphabetizes stories and I would probably look for a Sections story under "S" before "W".